x11-wm/pekwm/pekwm-0.2.1.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib desktop optfeature
```

Comment:
One thing I didn't spot earlier: Why enable multilib? I can't think of a reason for that.

All in all this looks good apart for few stylistic issues but I can fix those. Let me know if you consider this finished (+ the multilib issue).

Reply:
Yes, you're right. This is a result of my experiments. To be honest, when I was playing with it, the pure cmake didn't work for me, while cmake-multilib did. It's obvious that it was for a different reason(s) then, but I assumed that cmake-multilib is better supported, even if multilib doesn't make sense for a user app. However, it doesn't make any harm, so I let it in a multilib mode. My fault, I was wrong. It's corrected now via b6be44e.

Reply:
And I would appreciate if you could correct any stylistic issues.